storybooks/storybook
# CLI: Provide Typescript templates for Preact

Closes #



## What I did

The CLI only contained JS templates for Preact. This PR adds Typescript (and 4.9) to it.

## How to test

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template preact-vite/default-ts`
2. It should contain typescript stories


## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
